### ISSUE TITLE
Support for EmojiOne/Unicode 8.0

### ISSUE THREAD

**Issue by FabianOvrWrt** [enwin/emoji#8]
Newer versions of Firefox OS have the Firefox Emoji font, but in rooted devices you can replace it with EmojiOne (http://emojione.com/).

It would be nice to have support for the new emoji, because this app works way better than the system emoji keyboard on 2.6

**Comment by enwin** [comment]
Sorry I didn't saw the notification until today.

That looks like something easy to do but I'll might need your help.
Did you replace the existing emoji font of FirefoxOs or modify a css to display the EmojiOne one? If I understand correctly the whole FirefoxOS displays the EmojiOne font?

Thanks for your feedback!

**Comment by FabianOvrWrt** [comment]
Yes. I replaced the FirefoxEmoji font with EmojiOne. It can be done in any version of Firefox OS (replacing NotoColorEmoji), you just need to have root access.

**Comment by enwin** [comment]
Ok. I'm gonna add a switch in the settings to enable the unicode 8 range. That should do the trick.

**Comment by FabianOvrWrt** [comment]
That's great. Thank you!
